docs/src/inputs/radio/radio.jsx

```diff
@@ -1,7 +1,6 @@
 import {
```

Comment:
I imagine some of the further Documentation modernization is fodder for another day? :-)

Reply:
Yeah. I am hoping for 10.0.0 to take care of that.

Reply:
Resolving.